components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java

```diff
@@ -314,7 +315,15 @@ public long getTimestamp(String date) {
     public List<KeyManagerConfigurationDTO> getKeyManagerConfigurationsByOrganization(String organization)
             throws APIManagementException {
 
-        KeyMgtRegistrationService.registerDefaultKeyManager(organization);
+        // For Choreo scenario (Choreo organization uses the same super tenant Resident Key Manager
+        // Hence no need to register the default key manager per organization)
+        String givenOrganization = organization;
```

Comment:
the givenOrganization and organization is confusing, shall we make one of it as associated tenantDomain
it would be more clear

Reply:
Sure. Fixed via 419bad1d1bac6487be3fcdfdd827331c583dde5f